TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Management/AssetController.php

```diff
@@ -132,9 +132,9 @@ public function deleteAction(\TYPO3\Media\Domain\Model\Asset $asset)
     {
         try {
             $this->assetService->getRepository($asset)->remove($asset);
-            $this->addFlashMessage('assetHasBeenDeleted', '', Message::SEVERITY_OK, [$asset->getLabel()], 1412375050);
+            $this->addFlashMessage('Asset "%s" has been deleted.', '', Message::SEVERITY_OK, [$asset->getLabel()], 1412375050);
         } catch (AssetServiceException $exception) {
-            $this->addFlashMessage('media.deleteRelatedNodes', '', Message::SEVERITY_WARNING, [], 1412422767);
+            $this->addFlashMessage('Asset could not be deleted, because there are still Nodes using it', '', Message::SEVERITY_WARNING, [], 1412422767);
```

Comment:
this translation actually belongs to Neos and not Media, which is the problem with this approach and why it was done like it is originally..

same goes for the translation 1462308179 in line 164, which is even missing from the xlf files..

in these case the translation should be in Neos and the flash message should use `TYPO3.Neos` as package instead.. not sure how to deal with this tbh.